corehq/apps/case_importer/templates/case_importer/excel_config.html

```diff
@@ -20,7 +20,7 @@
       <p>
         Documentation for bulk case imports can be found on our
         <a
-          href="https://confluence.dimagi.com/display/commcarepublic/Updating+Cases+-+Update+multiple+case+types+in+a+single+Excel+import"
+          href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143946828/Importing+Cases+Using+Excel"
```

Comment:
More specific: https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143946828/Importing+Cases+Using+Excel#Update-Multiple-Case-Types-in-a-Single-Import